src/app/static/src/app/components/versions/VersionHistory.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+    <div v-if="versions.length > 0">
+        <h5 v-translate="'versionHistory'"></h5>
+        <div id="headers" class="row font-weight-bold pt-2">
+            <div class="col-md-1 header-cell"></div>
+            <div class="col-md-3 header-cell" v-translate="'versionName'"></div>
+            <div class="col-md-3 header-cell" v-translate="'lastUpdated'"></div>
+        </div>
+        <hr/>
+        <div v-for="v in versions">
+            <div :id="`v-${v.id}`"  class="row">
+                <div class="col-md-1 version-cell">
+                    <button v-b-toggle="`snapshots-${v.id}`" class="btn btn-xs bg-transparent shadow-none">
+                        <chevron-right-icon size="20" class="icon when-closed"></chevron-right-icon>
+                        <chevron-down-icon size="20" class="icon when-open"></chevron-down-icon>
+                    </button>
+                </div>
+                <div class="col-md-3 version-cell">
+                    {{v.name}}
+                </div>
+                <div class="col-md-3 version-cell">{{format(v.snapshots[0].updated)}}</div>
+            </div>
+            <b-collapse :id="`snapshots-${v.id}`">
+                <div v-for="s in v.snapshots" :id="`s-${s.id}`" class="row font-italic bg-light pb-3">
```

Comment:
Generally I think this looks and works great. There's some slight padding weirdness - this should probably be `py-2` rather than `pb-3` so it's even top and bottom, the same `py-2` added to each `version-cell` (line 12), and then `py-0` on the button toggle (line 13) so that it doesn't add extra vertical padding